src/Network/WebSockets/TSSocketManager.m

```diff
@@ -30,11 +26,11 @@ @interface TSSocketManager ()
 
 @property (nonatomic, readonly, strong) OWSSignalService *signalService;
 
-@property (nonatomic, retain) NSTimer *pingTimer;
-@property (nonatomic, retain) NSTimer *reconnectTimer;
```

Comment:
`retain` is the default behavior for objects, so this is redundant.  99% of the time I use `@property (nonatomic)`. The exceptions are when I make a property `weak` or `readonly`.